src/platform/forms-system/src/js/review/ObjectField.jsx

```diff
@@ -122,6 +122,7 @@ class ObjectField extends React.Component {
         if (objectFields.length > 1 && visible.length > 0) {
           return objectFields.filter(showField).map(renderField);
         }
+        // eslint-disable-next-line sonarjs/no-extra-arguments
```

Comment:
Since it's not clear why the function is taking 2 params instead of 1, this rule was disabled to be in compliance with the activation of the rule. I disabled it to prevent that the changes might break the existing form system.